Shift3/boilerplate-client-react
# Added `settings` tab to `updateProfilePage`

## Changes
1. Removed `theme` settings from `BitwiseNavbar`
2. Added `App Settings` tab to `UserProfilePage.tsx`.
3. Added `dark mode` and `language` preferences to `App Settings`.
4. Fixed `bug` in toggle of dark/light mode.

## Purpose
Based on our discussion with Design, the settings/theme should be under the UserProfilePage for better UX.

## Testing
Make sure to run `dj_starter` on the backend.

1. Pull in the changes to your local copy of this branch and run `yarn start`.
2. Navigate through the tabs in the UserProfilePage and make sure you can adjust the themes.

## Screenshots
![Screen Shot 2023-01-10 at 3 23 58 PM](https://user-images.githubusercontent.com/59614789/211654509-477f3446-7d57-4fc1-916a-e9f37e362c43.png)
